R/R/install.R

```diff
@@ -0,0 +1,90 @@
+#' Install Metaflow Python package
+#'
+#' This function wraps installation functions from [reticulate][reticulate::reticulate] to install the Python packages
+#' **metaflow** and it's Python dependencies.
+#'
+#' This package uses the [reticulate][reticulate::reticulate] package
+#' to make an interface with the [Metaflow](https://metaflow.org/)
+#' Python package.
+#'
+#' @param method `character`, indicates to use `"conda"` or `"virtualenv"`.
+#' @param envname `character`, name of environment into which to install.
```

Comment:
one remaining issue here is `envname`: if a user specifies a custom envname, onLoad() won't pick it up. There are a few ways to go about supporting this but not sure if adding the custom environment is really necessary since that can be handled via the `METAFLOW_PYTHON` variable. 

@savingoyal @jasonge27